compiler/rustc_privacy/src/lib.rs

```diff
@@ -1862,6 +1878,29 @@ impl SearchInterfaceForPrivateItemsVisitor<'_> {
             }
         }
 
+        let Some(effective_vis) = self.effective_vis else {
+            return false;
+        };
+        let reexported_ev = *effective_vis.at_level(Level::Reexported);
```

Comment:
```suggestion
        let reachable_at_vis = *effective_vis.at_level(Level::Reachable);
```
That's what the RFC says at least.
Were there any issues with this formulation?